src/SelfService/Infrastructure/Persistence/ECRRepositoryRepository.cs

```diff
@@ -33,9 +33,10 @@ public async Task AddRange(List<ECRRepository> ecrRepositories)
         await _dbContext.ECRRepositories.AddRangeAsync(ecrRepositories);
     }
 
-    public void RemoveRangeWithRepositoryName(List<string> repositoryNames)
+    [TransactionalBoundary]
```

Comment:
not sure why this was added back?

Reply:
Not sure either; Eager merging is my guess